README.md

```diff
@@ -1,5 +1,33 @@
+# Saturn UI
 This project was bootstrapped with [Create React App](https://github.com/facebookincubator/create-react-app).
 
 Guide [here](https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md).
 
 Dev is up [here](https://bvdp-saturn-dev.appspot.com/).
+
+Builds/deploying handled by CircleCI.
+
+
+### Developing
+
+1. We use node@8 (the current LTS). On Darwin with Homebrew:
```

Comment:
Just FYI, this is one of the more compelling reasons for using Docker. What if I want to work on this in my free time, but on my main project we are tied to node@6? Of course there are ways to accomplish the task, but Docker gives us a consistent answer on any project.